php/values.yaml

```diff
@@ -308,6 +308,15 @@ fpm:
     displayErrors: off
     errorReporting: E_ALL
     exposePhp: off
+    newrelic.enabled: true
+    newrelic.appname: ${NEWRELIC_PHP_APPNAME}
+    newrelic.license: ${NEWRELIC_PHP_LICENSE}
+    newrelic.logfile: /var/log/newrelic/php_agent.log
+    newrelic.loglevel: info
+    newrelic.browser_monitoring.attributes.enabled: true
+    newrelic.error_collector.record_database_errors: true
+    newrelic.browser_monitoring.auto_instrument: true
+    newrelic.transaction_tracer.detail: 1
```

Comment:
@cw-niinuma 
Why is PHP chart fixing it?
PHP chart is not dependent on newrelic-php-agent chart. Therefore, this fix is inappropriate.

Reply:
Isn't it written in helmfile values instead of php chart values?
php chart is design for stand-alone.